openssl-sys/build/find_normal.rs

```diff
@@ -239,3 +240,45 @@ fn execute_command_and_get_output(cmd: &str, args: &[&str]) -> Option<String> {
     }
     return None;
 }
+
+
+
+/// find openssl path on macport
+fn get_macport_openssl() -> Option<std::string::String> {
+    let out = Command::new("port")
+        .arg("-q")
+        .arg("installed")
+        .arg("openssl")
+        .output();
+    if out.is_ok() {
+        let res = out.ok().unwrap();
+        let outputs = std::str::from_utf8(&res.stdout).unwrap();
+        let version = get_macport_openssl_version(outputs);
+        
+        if version.is_some()
+            && version.unwrap() >= std::string::String::from("@1.1") {
```

Comment:
Trying to compare versions as strings seems suspicious.

Reply:
I revised some lines to compare version rather than string. I chose the way to write simple version struct for mac port openssl rather than rust semver. Semver did not parse mac port openssl version like "1.1.1h".